thankful/index.pug

```diff
@@ -31,7 +22,15 @@ html
 
 
                 hr
-                include /templates/email-signup.pug
+                h5.text-center Get started!
+                p.text-center
+                  | Are you a content creator or a donor?
```

Comment:
I think I prefer 'supporter' over 'donor', any other alternatives? (I really don't like 'donor')

Reply:
I just opened #3 instead, will merge this in the meantime.